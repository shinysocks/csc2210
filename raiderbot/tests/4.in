+--------------------+
|    **# ########    |
|  # ** #*    *      |
| #  ** =*...*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    # *|
|    #     ### # #  *|
|  ##        S       |
|             ###### |
+--------------------+
19 9 
n
wnee
n
n
w

