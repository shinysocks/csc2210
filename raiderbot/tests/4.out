Initial state: Robot at 19, 9 (0 gold)
+--------------------+
|    **# ########    |
|  # ** #*    *      |
| #  ** =*...*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    # *|
|    #     ### # #  *|
|  ##        S       |
|             ######R|
+--------------------+
========================================
Executing n
Robot at 19, 8 (0 gold)
+--------------------+
|    **# ########    |
|  # ** #*    *      |
| #  ** =*...*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    # *|
|    #     ### # #  *|
|  ##        S      R|
|             ###### |
+--------------------+
Executing wnee

Error: robot with current status of Robot at 19, 7 (1 gold) broke while executing wnee
+--------------------+
|    **# ########    |
|  # ** #*    *      |
| #  ** =*...*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    # *|
|    #     ### # #  R|
|  ##        S       |
|             ###### |
+--------------------+
========================================
